content/working_with/manager/ldap-integration.md

```diff
@@ -6,56 +6,177 @@ weight: 1450
 ---
 {{< param product_name >}} provides a user management mechanism, so you can define different users with different permissions, and upon login perform authentication and authorization to control the users’ access to resources. 
 
-The users can be either defined and managed in {{< param product_name >}} itself, or you can configure your Manager to integrate with an LDAP-based user-management system.
-You must select one of these options, as you cannot do both, and you must configure your manager accordingly upon installation or immediately afterwards, when no actions were performed on it yet.  
+The users can be either defined and managed in {{< param product_name >}} itself, or you can configure your Manager to integrate with an LDAP-based user-management system. You must select one of these options, as you cannot do both.
+
+If LDAP integration is enabled, {{< param product_name >}} system role membership should be configured using [user-groups]({{< relref "cli/maint_cli/usergroups.md" >}}).
+
+If you enable LDAP with existing users already on the manager those users will continue to exist but cannot be used to log in unless they match an LDAP username.
+e.g. if a user jbloggs exists on the manager before ldap is enabled then after LDAP is enabled a login is made by LDAP user jbloggs, the LDAP user jbloggs will be shown as the creator of any entities (e.g. blueprints, secrets, deployments) that were created by the original jbloggs user.
+
+The initial admin user (by default called 'admin') will still be authenticated using local authentication.
+
 User management can be performed using the CLI or the {{< param cfy_console_name >}}.
 
+Note that for Active Directory currently only one domain is supported- attempting to log in with any other domains in a forest is unsupported.
+
 {{% tip title="User Management Credentials" %}}
 You must have {{< param cfy_manager_name >}} administrator permissions to perform user-management related actions.
 {{% /tip %}}
 
-## Managing users by Integrating with an LDAP System
-If you choose to integrate with an external user-management system, make sure your manager is configured accordingly:
+## Basic integration
+In order to enable LDAP user integration you must know:
+* The LDAP server address, e.g. ldap://192.0.2.1 or ldaps://192.0.2.4
```

Comment:
```suggestion

* The LDAP server address, e.g. ldap://192.0.2.1 or ldaps://192.0.2.4
```

Reply:
LDAP uppercasing done where relevant in upcoming commit.